### PR TITLE
Added link to broaden description on SSL sertificates

### PR DESCRIPTION
**Pull request by MatthewBellwood**
This link provides information about SSL/TLS Certificate, and later on in the post, you can find MySQL description also.

### CODE REVIEW

File: docs/en/00_Getting_Started/01_Installation/How_To/MySQL_SSL_Support.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ In a typical Silverstripe set up, you will only need to use a single host to fun
 
 In some cases, however, you may be required to connect to a database on a remote host. Connecting to a remote host without SSL encryption exposes your data to [packet sniffing](http://www.linuxjournal.com/content/packet-sniffing-basics) and may compromise the security of your Silverstripe instance.
 
-This article demonstrates how to generate SSL certificates using MySQL and implementing them in Silverstripe.
+This article demonstrates how to generate [SSL certificates](https://hosting.review/web-hosting-glossary/#12) using MySQL and implementing them in Silverstripe.
```
Comment (robbieaverill) [inline docs/en/00_Getting_Started/01_Installation/How_To/MySQL_SSL_Support.md]: Is there a particular reason you've chosen this resource to link to?

**Comment by MatthewBellwood** [comment]
Hello,

No particular reason just really liked it, it's thorough and well written,
I though it would fit well.

**Comment by robbieaverill** [comment]
Thanks!

**Comment by robbieaverill** [comment]
Sorry, I merged this a little early. I'd prefer a more authoritative resource if possible =)

**Pull request merged**